<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButtonOuvrirPort">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ouvrir le port série</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEnvoyai">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Envoyer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxChoixSerie">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Choisisez un port série</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>texte à envoyer :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Texte reçu :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTextEnvoyer">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditRecu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>471</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
